uui-components/src/navigation/MainMenu/Burger/Burger.tsx

```diff
@@ -99,9 +99,9 @@ export class Burger extends React.Component<BurgerProps, BurgerState> {
                     className={ cx(this.props.cx, uuiBurger.menu, css.container, this.state.isOpen && uuiBurger.menuOpen) }
                     { ...this.props.rawProps }
                 >
-                    <div className={ uuiBurger.button } onClick={ this.toggleBurgerMenu }>
+                    <button className={ uuiBurger.button } onClick={ this.toggleBurgerMenu }>
```

Comment:
Do we need to reset default styles from button here?

Reply:
Yeah, thanks fixed.